harvest/broker/dummy.py

```diff
@@ -125,30 +137,24 @@ def fetch_price_history(self,
         results = results.loc[(open_time < results.index.time) & (results.index.time < close_time)]
         results = results[(results.index.dayofweek != 5) & (results.index.dayofweek != 6)]
 
-        results.columns = pd.MultiIndex.from_product([[ticker], results.columns])
-
         return results.iloc[::-1]
 
-    def fetch_latest_stock_price(self) -> pd.DataFrame:
-        results = pd.DataFrame()
+    def fetch_latest_stock_price(self) -> Dict[str, pd.DataFrame]:
+        results = {}
         last = dt.datetime.now() - dt.timedelta(days=7)
         today = dt.datetime.now()
         for ticker in self.watch:
             if ticker[0] != '@':
-                result = self.fetch_price_history(last, today, self.interval, ticker).iloc[0]
-                results = results.join(result, how='outer')
-
-        results.columns = pd.MultiIndex.from_tuples(results.columns)
+                results[ticker] = self.fetch_price_history(last, today, self.interval, ticker).iloc[[0]]
         return results
         
-    def fetch_latest_crypto_price(self) -> pd.DataFrame:
-        results = pd.DataFrame()
+    def fetch_latest_crypto_price(self) -> Dict[str, pd.DataFrame]:
+        results = {}
```

Comment:
functions now return dictionaries instead of multi-index pandas dataframes

Reply:
Note that the individual DataFrames still has a multiIndex, as defined in the comments: https://github.com/tfukaza/harvest/blob/bb11b51e95060c6790120742f39af274fc18ffd6/harvest/broker/_base.py#L91-L94
The difference is that now they are returned as a dictionary rather than joining them.